collectd-docker/docs/cpu.throttling_data.throttled_periods.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Throttled Periods
+brief: Number of periods throttled
+metric_type: cumulative_counter
+---
+### Throttled Periods
+
+(Optional metric) A cumulative count of number of periods the container has been throttled for. If container attempts to use more than its quota, it will be throttled. Docker reports 0 if there is no quota constraint on the container.
```

Comment:
"A cumulative count of the number of periods the container has been throttled. If the container attempts to use more than its quota, it will be throttled. Docker reports 0 if there is no quota constraint on the container."